tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py

```diff
@@ -874,6 +876,84 @@ def crop_center(img, cropx, cropy, cropz):
 
         return img[startz:endz, starty:endy, startx:endx, :]
 
+class TransformedCropWithAutoScale(Preprocessor):
+    __provider__ = 'transformed_crop_with_auto_scale'
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'dst_width': NumberField(
+                value_type=int, optional=True, min_value=1, description="Width of heatmaps output."
+            ),
+            'dst_height': NumberField(
+                value_type=int, optional=True, min_value=1, description="Height of heatmaps output."
+            ),
+            'stride': NumberField(
+                value_type=int, optional=False,
+                description="Stride for network. It is input size of heatmaps / output size of heatmaps."
+            )
+        })
+
+        return parameters
+
+    def configure(self):
+        self.input_height, self.input_width = get_size_from_config(self.config)
```

Comment:
`get_size_from_config` assume also that `size` parameter is available. It is used when `dst_height` and `dst_width` are equal. Please add it for alignment with other preprocessors.